app/models/pet.rb

```diff
@@ -0,0 +1,8 @@
+class Pet < ApplicationRecord
+  validates_presence_of :name,
+                        :archetype,
+                        :breed
```

Comment:
I think breed might be a good candidate for an optional field? Let's leave it off of validates_presence_of.

Reply:
That's a great point, especially with how common mutts are. Removed in this last push.